### PR TITLE
WebSocketApp.run_forever(), read(): check for ssl.SSLEOFError; on exc…

### PR DESCRIPTION
**Pull request by bubbleboy14**
…eption, if custom_dispatcher, pass bool(reconnect) to handleDisconnect()

### CODE REVIEW

File: websocket/_app.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import ssl
```
Comment (engn33r) [inline websocket/_app.py:1]: Change this to `from ._ssl_compat import *` like the other files. _ssl_compat.py exists for the edge case where SSL can't be imported

**Comment by engn33r** [comment]
The change to specify the reconnecting bool value to `handleDisconnect` makes sense to me. I'd suggest a different approach for `import ssl` or the EOF error, though I'm not sure how exactly to replace it. It may need edits to _ssl_compat.py.

This finding shows that better test coverage of the different branches would be a good improvement to work on in the future :+1:

**Comment by bubbleboy14** [comment]
Good call @engn33r I made the suggested changes.

Just realized I did "from ._ssl_compat import SSLEOFError" instead of the recommended "from ._ssl_compat import *". I kind of like the single import more, but on the other hand maybe we should be checking for the other exceptions as well.

In either case, this feels like an improvement for now, and certainly addresses the edge case I encountered.

**Comment by engn33r** [comment]
Great, thanks for the fix!

**Pull request merged**